tt_eager/tt_dnn/op_library/topk/kernels/compute/topk.cpp

```diff
@@ -28,14 +28,22 @@ void MAIN {
     constexpr uint32_t logk = get_compile_time_arg_val(9);
     constexpr uint32_t logWt = get_compile_time_arg_val(10);
 
+    constexpr uint32_t input_dest_start = 0;
```

Comment:
can you add a small comment explaining these numbers

Reply:
Done